tests/e2e/driver/CheReporter.ts

```diff
@@ -86,8 +86,8 @@ class CheReporter extends mocha.reporters.Spec {
       if (TestConstants.TS_SELENIUM_RESPONSE_INTERCEPTOR) {
         CheApiRequestHandler.enableResponseInterceptor();
       }
-      await preferencesHandler.setConfirmExit(AskForConfirmationType.never);
-      await preferencesHandler.setTerminalType(TerminalRendererType.dom);
+      // await preferencesHandler.setConfirmExit(AskForConfirmationType.never);
```

Comment:
@makambalaji  Why were the preferences handler calls removed? It doesn't contact keycloak, that's purely a Theia thing.

Reply:
Ok, in that case that's not good. Do you know how are the API requests authorized without dedicated keycloak? These requests are CRITICAL for correct execution of tests.

Reply:
There is a possibility for authentication with Oauth_Proxy cookie
After immediate login, we can see this in the Request headers. Need to check from where we can fetch these details [copying from slack conversation], will work on this solution as part of github issue : https://github.com/eclipse/che/issues/21203